plotly/figure_factory/_dendrogram.py

```diff
@@ -288,12 +290,11 @@ def get_dendrogram_traces(self, X, colorscale, distfun, linkagefun, hovertext):
             hovertext_label = None
             if hovertext:
                 hovertext_label = hovertext[i]
-            trace = dict(
-                type='scatter',
+            trace = graph_objs.Scatter(
                 x=np.multiply(self.sign[self.xaxis], xs),
                 y=np.multiply(self.sign[self.yaxis], ys),
                 mode='lines',
-                marker=dict(color=colors[color_key]),
+                marker=graph_objs.Marker(color=colors[color_key]),
```

Comment:
Stick with `dict` or use `graph_objs.scatter.Marker`.  `graph_objs.Marker` is deprecated in version 3